File: stage1/problems/problem18/starter.py
import sys

def solve():
  """
  Reads input from stdin, solves the problem, and prints the output.
  """
  try:
    # --- Input reading ---
    list_a = list(map(int, sys.stdin.readline().split()))
    list_b = list(map(int, sys.stdin.readline().split()))
    
    # --- Solution ---
    # The following variables are available:
    # list_a: list of ints
    # list_b: list of ints
    
    # TODO: Implement the solution
    
  except (ValueError, IndexError):
    pass

if __name__ == "__main__":
    solve()

File: stage1/problems/problem19/starter.java
import java.util.*;

public class starter {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        // --- Input reading ---
        String[] tokens = sc.nextLine().split(" ");
        int[] nums = new int[tokens.length];
        for(int i = 0; i < tokens.length; ++i) {
            nums[i] = Integer.parseInt(tokens[i]);
        }

        // --- Solution ---
        // The following variables are available:
        // nums: int[]

        // TODO: Implement the solution
        
        sc.close();
    }
}
